.evergreen/auth_aws/README.md

```diff
@@ -1,4 +1,64 @@
 # Configuration Scripts for End-to-end Testing
 
-These scripts were taken from [mongo-enterprise-modules](https://github.com/10gen/mongo-enterprise-modules/tree/master/jstests/external_auth_aws) 
-and intended to simplify creating users, attaching roles to existing EC2 instances, launching an Amazon ECS container instance, etc.
\ No newline at end of file
+These scripts were originally taken from [mongo-enterprise-modules](https://github.com/10gen/mongo-enterprise-modules/tree/master/jstests/external_auth_aws) 
+and intended to simplify creating users, attaching roles to existing EC2 instances, launching an Amazon ECS container instance, etc.
+
+## Test Process
+
+For all testing variants except for ECS, the general test flow is:
+
+- Set up the required secrets for the the test variant:
+
+```bash
+cd $DRIVERS_TOOLS/.evergreen/auth_aws
+# Fetch the secrets from the vault and write to a local `secrets-export.sh` file.
+bash setup_secrets.sh drivers/aws_auth
+```
+
+See https://wiki.corp.mongodb.com/display/DRIVERS/Using+AWS+Secrets+Manager+to+Store+Testing+Secrets for more background
+on how the secrets are managed.
+
+- Run the setup for the test variant and then run your specific tests.
+
+```bash
+cd $DRIVERS_TOOLS/.evergreen/auth_aws
+# Create a python virtual environment.
+cd ./activate-authawsvenv.sh
+# Source the environment variables.
+. aws_setup.sh <variant>
+# Configure the environment and the server.
+python aws_tester.py <variant>
+# Run your driver-specific tests here.
+```
+
+It is recommended that these actions be broken into scripts that can be run locally as well as in CI.  The instructions
```

Comment:
Do we already have these broken up into scripts? If so we could link to them here as an example.

Reply:
We do not, no, that is part of a broader effort and what the EVG docs recommend.

Reply:
Is it worth opening a ticket to track that work?

Reply:
https://jira.mongodb.org/browse/PYTHON-4069